projects/core/src/store-finder/model/index.ts

```diff
@@ -1,4 +1,7 @@
 export * from './location';
```

Comment:
just a side-question: can we store every model/interface in one file? so we don't have to reimport each model in other models etc?

Reply:
From my point of view i'm not sure if it is good idea. 
If we put every interface into one file it will be quite long and less readable.